packages/strapi-design-system/src/MainNav/MainNav.js

```diff
@@ -1,23 +1,26 @@
 import React from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
-import { Grid } from '../Grid';
+import { Flex } from '../Flex';
 import { MainNavContext } from './MainNavContext';
 
-const MainNavWrapper = styled(Grid)`
+const MainNavWrapper = styled(Flex)`
   width: ${({ condensed }) => (condensed ? 'max-content' : `${224 / 16}rem`)};
   background: ${({ theme }) => theme.colors.neutral0};
   position: sticky;
   top: 0;
   height: 100vh;
+  overflow-y: auto;
+  /* That doesn't work, overflow-y auto still hides the condensed toggle button when on condensed state */
+  overflow-x: visible;
```

Comment:
I didn't find a fix for this yet, the `overflow-y: auto` or `scroll` (tried both) causes overflow-x to be hidden which we don't want for `NavCondense` and `notifications` UI:
<img width="367" alt="Screenshot 2023-01-13 at 11 08 02" src="https://user-images.githubusercontent.com/71838159/212297098-9e864537-87c5-443d-8e7e-1619cfd94d9b.png">

`overflow-x: visible;` do not fix this issue

(cc @joshuaellis @gu-stav)

Reply:
Sorry I'm not familiar with the names you're using – you're saying the NEW flag is being cut off right?

Reply:
oh yeah sorry, the "new" badge and also the button toggle next to the user name (it acts like the `MainNav` `overflow-y: auto` means `overflow hidden`)
<img width="101" alt="image" src="https://user-images.githubusercontent.com/71838159/212370665-1c90df2e-cfd1-4ec7-9fd9-88a3704d866d.png">